W-20897943-mcp-server-instances-fa (#634)
mulesoft/docs-api-manager: modules/ROOT/pages/create-instance-public-exchange.adoc

= Adding an MCP Server from Public Exchange
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> This workflow describes how to create an MCP server in API Manager by selecting an asset published in the Public Exchange (Provided by MuleSoft).

Reviewer: ```suggestion
This workflow describes how to create an MCP server in API Manager by selecting an asset published in the public Exchange (Provided by MuleSoft).
```

> You can use this workflow when you want to base a new instance on an MCP server published to Exchange and maintained outside your organization.

Reviewer: ```suggestion
To base a new instance on an MCP server published to Exchange and maintained outside your organization, use this workflow.
```

== Before You Begin

> Ensure that you have the required permissions to create instances in API Manager and access assets in Exchange.

Reviewer: ```suggestion
Make sure you have the required permissions to create instances in API Manager and access assets in Exchange.
```

Reviewer: Can you list the required permissions as well?

== Add an MCP Server or Agent Instance from Public Exchange

. Navigate to *Anypoint Platform* > *API Manager*.

> . Depending on the instance type you want to create, do one of the following:

Reviewer: ```suggestion
. Select an instance type.
```

> * To create an MCP server instance, click *MCP Server*.

Reviewer: Delete this line

. Click *Add* to start the guided experience.
. Click *Next* until you reach the *Select asset from Exchange* step.

. In *Organization*, select *Provided by MuleSoft*.
+
Selecting *Provided by MuleSoft* lets you browse and search assets that are publicly available in Exchange.

> . Optional: Filter assets by tag.

Reviewer: ```suggestion
. To find an asset, search by name or filter using the *Provided by Mulesoft* tag.
```

+

> Use tags to narrow down the list of available assets. Tag filtering is supported when creating MCP server instances and helps you quickly find relevant public assets.

Reviewer: I think you can delete this line because line 26 explains this.

. Select an asset from the list.
+
The asset list shows the latest available version and the date it was published to Exchange. You can select a different version if needed.

> . Optional: Click *View in Exchange* to review the asset details before continuing.

Reviewer: ```suggestion
. To review the asset details before continuing, click *View in Exchange*.
```
